src/lib/web/routes/v1/rest/mavlink.rs

```diff
@@ -1,11 +1,17 @@
 use std::net::SocketAddr;
 
+use crate::{
+    drivers::rest::parse_query,
+    mavlink_json::{MAVLinkJSON, MAVLinkJSONHeader},
+};
```

Comment:
We are putting the `crate::` imports after the 3rd party, separated by a blank line, as in:

```
use std::sync::Arc;

use anyhow::Result;

use crate::callbacks::{Callbacks, MessageCallback};
```